src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -1,31 +1,42 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
-    <span class="pficon pficon-ok" tooltip="Everything looks okay" placement="top"></span>
-    <deployments-donut [applicationId]="applicationId" [mini]="true" ></deployments-donut>
-    <span id="versionLabel">{{ version | async }}</span>
-    <div class="pull-right dropdown-kebab-pf dropdown" dropdown (click)="$event.stopPropagation()">
-      <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true"
-        aria-expanded="true" data-toggle="dropdown" dropdownToggle>
-        <span class="fa fa-ellipsis-v"></span>
-      </button>
-      <ul class="dropdown-menu-right dropdown-menu" aria-labelledby="kebab" role="menu" *dropdownMenu>
-        <li>
-          <a>View logs</a>
-        </li>
-        <li>
-          <a>View OpenShift Console</a>
-        </li>
-        <li role="separator" class="divider"></li>
-        <li>
-          <a>Open Application</a>
-        </li>
-        <li role="separator" class="divider"></li>
-        <li>
-          <a>Delete</a>
-        </li>
-      </ul>
+    <div class="row mini-card-content">
+      <div class="col-xs-2">
+        <span class="pficon fa-lg pficon-ok" tooltip="Everything looks okay" placement="top"></span>
+      </div>
+      <div class="col-xs-7">
+        <deployments-donut [applicationId]="applicationId" [mini]="true"></deployments-donut>
+      </div>
+      <div class="col-xs-1">
+        <span id="versionLabel">{{ version | async }}</span>
+      </div>
+      <div class="col-xs-2">
+        <div class="dropdown-kebab-pf dropdown" (click)="$event.stopPropagation()" dropdown>
+          <button class="btn btn-link pull-right dropdown-toggle" id="kebab" type="button" aria-haspopup="true" aria-expanded="true"
+            data-toggle="dropdown" dropdownToggle>
+            <span class="fa fa-ellipsis-v"></span>
+          </button>
+          <ul class="dropdown-menu-right dropdown-menu deployment-dropdown" aria-labelledby="kebab" role="menu" *dropdownMenu>
+            <li>
+              <a>View logs</a>
+            </li>
+            <li>
+              <a>View OpenShift Console</a>
+            </li>
+            <li role="separator" class="divider"></li>
+            <li>
+              <a>Open Application</a>
+            </li>
+            <li role="separator" class="divider"></li>
+            <li>
+              <a>Delete</a>
+            </li>
+          </ul>
+        </div>
+      </div>
     </div>
     <div [collapse]="collapsed">
+      <hr />
```

Comment:
Controlling spacing with a `<hr />` feels wrong. @mindreeper2420 are you OK with this?

Reply:
It's not meant to be adding spacing, it's for the grey line that appears between the "collapsed" state content and the "expanded" content.

Reply:
This is the correct usage of this tag. You can remove the trailing slash though - that is not needed `<hr>`

Reply:
oh, I missed the grey line. Thanks.